source/Person.cpp

```diff
@@ -16,6 +16,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "Person.h"
 
 #include "DataNode.h"
+#include "FormationPattern.h"
```

Comment:
I'm not sure, but I think you don't need to include this in this case.
```suggestion
```

Reply:
It complains about an incomplete type on line 39 if I leave this include out. (The include is apparently needed for getting the pattern from gamedata.)

Reply:
ok